ui/core/components/inputs/buffs_debuffs.ts

```diff
@@ -0,0 +1,570 @@
+import { PickerStatOptions } from "./stat_options";
+
+import * as InputHelpers from '../input_helpers';
+
+///////////////////////////////////////////////////////////////////////////
+//                                 RAID BUFFS
+///////////////////////////////////////////////////////////////////////////
+
+// TODO: Classic buff icon by level
+export const AllStatsBuff = withLabel(
+	makeTristateRaidBuffInput({id: ActionId.fromSpellId(9885), impId: ActionId.fromSpellId(17055), fieldName: 'giftOfTheWild'}),
+	'Mark of the Wild',
+);
+
+// Separate Strength buffs allow us to use a boolean pickers for Horde specifically
+export const AllStatsPercentBuffAlliance = InputHelpers.makeMultiIconInput([
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(20217), fieldName: 'blessingOfKings', faction: Faction.Alliance}),
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(409580), fieldName: 'aspectOfTheLion', faction: Faction.Alliance}),
+], 'Stats %');
+
+export const AllStatsPercentBuffHorde = withLabel(
+	makeBooleanIndividualBuffInput({id: ActionId.fromSpellId(409580), fieldName: 'aspectOfTheLion', faction: Faction.Horde}),
+	'Stats %',
+);
+
+// TODO: Classic armor buff ranks
+export const ArmorBuff = InputHelpers.makeMultiIconInput([
+	makeTristateRaidBuffInput({id: ActionId.fromSpellId(10293), impId: ActionId.fromSpellId(20142), fieldName: 'devotionAura'}),
+	makeBooleanRaidBuffInput({id: ActionId.fromItemId(1478), fieldName: 'scrollOfProtection'}),
+], 'Armor');
+
+export const StaminaBuff = InputHelpers.makeMultiIconInput([
+	makeTristateRaidBuffInput({id: ActionId.fromSpellId(10938), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude'}),
+	// makeTristateRaidBuffInput({id: ActionId.fromSpellId(10937), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 48, maxLevel: 59}),
+	// makeTristateRaidBuffInput({id: ActionId.fromSpellId(2791), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 36, maxLevel: 47}),
+	// makeTristateRaidBuffInput({id: ActionId.fromSpellId(1245), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 24, maxLevel: 35}),
+	// makeTristateRaidBuffInput({id: ActionId.fromSpellId(1244), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 12, maxLevel: 23}),
+	// makeTristateRaidBuffInput({id: ActionId.fromSpellId(1243), impId: ActionId.fromSpellId(14767), fieldName: 'powerWordFortitude', minLevel: 1, maxLevel: 11}),
+	makeTristateRaidBuffInput({id: ActionId.fromSpellId(11767), impId: ActionId.fromSpellId(18696), fieldName: 'bloodPact'}),
+	makeBooleanRaidBuffInput({id: ActionId.fromItemId(10307), fieldName: 'scrollOfStamina'}),
+], 'Stamina');
+
+// Separate Strength buffs allow us to use boolean pickers for each
+export const StrengthBuffAlliance = withLabel(
+	makeBooleanRaidBuffInput({id: ActionId.fromSpellId(425600), fieldName: 'hornOfLordaeron', faction: Faction.Alliance}),
+	'Strength',
```

Comment:
Would name this "Str/Agi" or equivalent since it provides both stats.

Reply:
Actually now that you've separated it out by faction anyway, it would be even better to put Might and Horn in their own field, since they're exclusive with one another.

Reply:
Oh nice I didn't realize that. I can do that then 👍